test/coordinator-test.js

```diff
@@ -0,0 +1,19 @@
+import { assert } from 'chai';
+import os from 'os';
+import sinon from 'sinon';
+import { getWorkerCount } from '../lib/coordinator';
+
+describe('coordinator', () => {
+  it('uses the correct number of cpus', () => {
+    const sandbox = sinon.sandbox.create();
+    sandbox.stub(os, 'cpus').returns([{}, {}, {}, {}, {}]);
```

Comment:
it'd be nice for the test if this array of 5 objects was stored in a variable, so that its length could be used in place of a hardcoded `4`